terraform/jenkins-master/configure-master/jenkins-master.groovy

```diff
@@ -0,0 +1,108 @@
+// Paste this script in Jenkins Script Console to configure Azure VM plugin
+import com.microsoft.azure.vmagent.builders.*;
+
+def bionicTemplate = new AzureVMTemplateBuilder()
+    .withName("accbionic")
+    .withLabels("ACC-1804")
+    .withLocation(AGENTS_LOCATION)
+    .withVirtualMachineSize(VM_SIZE_SGX)
+    .withExistingStorageAccount(AGENTS_STORAGE_ACCOUNT)
+    .withUsageMode("Only build jobs with label expressions matching this node")
+    .addNewAdvancedImage()
+        .withCustomManagedImage(AGENTS_IMAGE_PREFIX + "acc-ubuntu-18.04")
+        .withInitScript("sudo gpasswd -a ${ADMIN_USER} docker \n" +
+                        "sudo chmod g+rw /var/run/docker.sock")
+    .endAdvancedImage()
+    .withAdminCredential(CREDENTIALS_ID_ADMIN)
+    .build();
+
+def xenialTemplate = new AzureVMTemplateBuilder()
+    .withName("accxenial")
+    .withLabels("ACC-1604")
+    .withLocation(AGENTS_LOCATION)
+    .withVirtualMachineSize(VM_SIZE_SGX)
+    .withExistingStorageAccount(AGENTS_STORAGE_ACCOUNT)
+    .withUsageMode("Only build jobs with label expressions matching this node")
+    .addNewAdvancedImage()
+        .withCustomManagedImage(AGENTS_IMAGE_PREFIX + "acc-ubuntu-16.04")
+        .withInitScript("sudo gpasswd -a ${ADMIN_USER} docker \n" +
+                        "sudo chmod g+rw /var/run/docker.sock")
+    .endAdvancedImage()
+    .withAdminCredential(CREDENTIALS_ID_ADMIN)
+    .build();
+
+
+def ubuntunonSGXTemplate = new AzureVMTemplateBuilder()
+    .withName("ubuntunonsgx")
+    .withLabels("nonSGX")
+    .withLocation(AGENTS_LOCATION)
+    .withVirtualMachineSize(VM_SIZE_NONSGX)
```

Comment:
I don't believe this sizing is correct, I believe we actually needed a "Standard_F8s_v2". This caused tls issues.